tests/test_nested_mapping.py

```diff
@@ -166,9 +172,9 @@ def test_recursive_update_overwrites_string_with_string(self):
 
 class TestRepresentation:
     def test_str_conversion(self, nested_nestmap):
-        desired = ("NestedMapping contents:\n├─foo: 5\n├─bar: \n│ ├─bogus: "
-                   "\n│ │ ├─a: 42\n│ │ └─b: 69\n│ └─baz: meh\n├─moo: "
-                   "yolo\n└─yeet: \n  ├─x: 0\n  └─y: 420")
+        desired = ("NestedMapping contents:\n├─bar: \n│ ├─bogus: \n│ │ ├─a: "
+                   "42\n│ │ └─b: 69\n│ └─baz: meh\n├─yeet: \n│ ├─x: 0\n│ └─y: "
+                   "420\n├─foo: 5\n└─moo: yolo")
```

Comment:
Perhaps write this out as a triple-quoted string? Because now I still have no idea what this will look like
```suggestion
        desired = """
NestedMapping contents:
├─bar: 
│ ├─bogus: 
│ │ ├─a: 42
│ │ └─b: 69
│ └─baz: meh
├─yeet: 
│ ├─x: 0
│ └─y: 420
├─foo: 5
└─moo: yolo
""".strip()
```

Reply:
Certainly a useful idea in this case 👍